android/src/main/kotlin/com/example/user_location_plugin/UserLocationPlugin.kt

```diff
@@ -19,13 +19,18 @@ import io.flutter.plugin.common.MethodChannel
 import io.flutter.plugin.common.MethodChannel.MethodCallHandler
 import io.flutter.plugin.common.MethodChannel.Result
 import io.flutter.plugin.common.PluginRegistry
+import com.google.android.gms.location.*
```

Comment:
avoid using wildcard imports, use individual imports

Reply:
Android studio do that automatically after i import everything i need, maybe because i use the whole library.

Reply:
You can configure that on AS, to avoid wildcard import

Reply:
there is a configuration on android studio to stop it from make that automatically, check this post: https://intellij-support.jetbrains.com/hc/en-us/community/posts/206203659-Turn-off-Wildcard-imports-

Reply:
or this one: https://www.jetbrains.com/help/idea/creating-and-optimizing-imports.html#disable-wildcard-imports